bin/labelcat.js

```diff
@@ -1,30 +1,32 @@
 #!/usr/bin/env node
 
 const util = require('../src/util.js');
-const settings = require('../settings.json'); // eslint-disable-line node/no-missing-require
+const settings = require('../settings.json');
 
 require(`yargs`)
   .demand(1)
   .command(
-    `retrieveIssues <repoDataFilePath> <issuesDataFilePath>`,
-    `Retrieves issues from a .txt file of gitHub repositories.`,
-    {},
+    `retrieveIssues <repoDataFilePath> <issuesDataFilePath> <label>`,
+    `Retrieves issues from a .txt file of gitHub repositories. Options: -a`,
+    function(command) {
+      command.options({alternatives: {alias: 'a'}});
```

Comment:
I think you need to add `type: 'array'` and `requiresArg: true` to the options object

Reply:
Also, need to update the README with the new instructions for running the command, and the test should test the `alternatives` stuff too.